site/content/integrate/apps/deployment/_index.md

```diff
@@ -52,40 +49,128 @@ For a go app the manifest snippet would look like this:
+    1. Check **I acknowledge that the current settings might result in this bucket and the objects within becoming public.**
+1. Select **Create Bucket**.
+
+##### 2. Create an IAM user access key and secret
+
+You will need an access key and secret so that `appsctl` can provision the app. These credentials can come from creating an IAM user, using a privileged IAM user, or even using the AWS account owners personal access key. Please follow the instructions [provided by AWS](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/) to complete these steps and save the `Access key ID` and `Secret access key` values.
+
+##### 3. Set AWS Provision environment variables
+
+Open a terminal where you installed the Apps plugin and set the following variables to the AWS credentials just created and saved:
+
+- `MM_APPS_PROVISION_AWS_ACCESS_KEY`  
+- `MM_APPS_PROVISION_AWS_SECRET_KEY`
+
+Set the following environment variables based on bucket name and region
+
+- `MM_APPS_S3_BUCKET`  
+- `MM_APPS_AWS_REGION`
+
+##### 4. Initialize the AWS resources
 
-### Deploying third-party apps to AWS
+The following command will create a Mattermost invoke user and security policy for use with AWS.
 
-Note that third-party apps are not supported in the Mattermost Cloud - they're used only for developer testing convenience. Provisioning in the third-party AWS cloud environment is done by the **appsctl** tool using the command:
+`go run ./cmd/appsctl aws init --create --create-access-key`
+
+The output of the command will contain two "Invoke" environment variables.
+
+##### 5. Set AWS Invoke environment variables
+
+The second set of credentials created from the above initialization step will be used by Mattermost to invoke Apps.  Set these variables in the location of your running Mattermost server repo.
+
+- `MM_APPS_AWS_ACCESS_KEY`
+- `MM_APPS_AWS_SECRET_KEY`
+
+#### Restart the Mattermost Server
+
+Ensure you have the following environment variables set.
+
+- `MM_APPS_AWS_ACCESS_KEY`
+- `MM_APPS_AWS_SECRET_KEY`
+- `MM_APPS_S3_BUCKET`
+- `MM_APPS_AWS_REGION`
+
+Restart the Mattermost server to complete your Mattermost and AWS setup. You can now deploy an app to AWS.
+
+#### Test your AWS configuration and permissions
+
+Run the following commands to test and validate your AWS configuration and permissions.
 
 ```bash
-go install github.com/mattermost/mattermost-plugin-apps/cmd/appsctl@latest
-appsctl provision app /PATH/TO/YOUR/APP/BUNDLE
+go run ./cmd/appsctl aws test lambda
+go run ./cmd/appsctl aws test s3
 ```
 
-It reads appropriate AWS credentials from environment variables:
+#### Optional: Provision the hello-lambda app
 
-`APPS_PROVISION_AWS_ACCESS_KEY`
+A test command is also available if you would like to test provisioning the hello-lambda test app. This will add a manifest to the S3 bucket and add a corresponding lambda function to AWS.
 
-`APPS_PROVISION_AWS_SECRET_KEY`
+Build hello-lambda bundle
```

Comment:
New section?

Reply:
Dylan and I discussed this too. I have this under the `Optional: Provision the hello-lambda app` subsection.  It is not necessary to provision this hello-lamda app, but it should be documented.  

I took your suggestion and completely pull the section out of `Setting up your Mattermost instance and AWS for deploying apps` section and added to its own section after `Deploying an app to AWS`

Reply:
see [`f674576` (#826)](https://github.com/mattermost/mattermost-developer-documentation/pull/826/commits/f674576da3f507a3167c8e12b1009bb7d952489f)